hedera-node/infrastructure/grafana/dashboards/production/hedera-node/network-details-prom.json

```diff
@@ -51,20 +51,40 @@
                   "index": 1,
                   "text": "ACTIVE"
                 },
-                "3": {
-                  "color": "orange",
-                  "index": 2,
-                  "text": "DISCONNECTED"
-                },
                 "4": {
                   "color": "yellow",
-                  "index": 3,
+                  "index": 2,
                   "text": "BEHIND"
                 },
                 "5": {
                   "color": "purple",
+                  "index": 3,
+                  "text": "FREEZING"
+                },
+                "6": {
+                  "color": "yellow",
                   "index": 4,
-                  "text": "MAINTENANCE"
+                  "text": "FREEZE COMPLETE"
+                },
+                "7": {
+                  "color": "yellow",
+                  "index": 5,
+                  "text": "OBSERVING"
+                },
+                "9": {
+                  "color": "yellow",
+                  "index": 6,
+                  "text": "CHECKING"
+                },
+                "10": {
+                  "color": "yellow",
+                  "index": 7,
+                  "text": "RECONNECT COMPLETE"
```

Comment:
Should this be something besides yellow? Like Green maybe? I don't have much context but just a thought.

Reply:
This was intentional, but I don't have strong feelings on it, but I'll provide my logic on having it be yellow.

During the reconnect process, a node may go through several reconnects. If the node is far enough behind when initiating a reconnect, it will take a long time to sync the state, so even when that initial reconnect finishes, the node may still find that it is behind. In those cases, it may initiate several rounds of reconnects. (in theory) Each time the reconnect will be shorter, and after several reconnects, it will be caught up enough to join consensus and go active once again.

This is the logic in leaving it yellow, as the node isn't truly "green" until it transitions to the active state.